guacamole/src/main/webapp/translations/cz.json

```diff
@@ -0,0 +1,885 @@
+{
+
+    "NAME" : "Čeština",
+
+    "APP" : {
+
+        "NAME" : "Apache Guacamole",
+        "VERSION" : "${project.version}",
```

Comment:
These two lines should be omitted - only the base translation file has these.

Reply:
Fixed now